Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[workspace]
+
+members = [
+    "lib/srv/desktop/rdp/rdpclient",
+    "lib/datalog/roletester"
+]
+
+[profile.dev]
+debug = 1
+lto = "off"
+
+[profile.release]
```

Comment:
Why are we not using lto for the release build? By default is off https://doc.rust-lang.org/cargo/reference/profiles.html#release
I think that `"thin"` is worth a try.

Reply:
This is something @xacrimon added a ways back. Joel, can you chime in?

Reply:
Yeah sure, so I added it since I mostly copied these build configs from a gist to get some better debugging. The reason LTO is off is that at least when I measured and tested this last it didn't matter too much for smaller projects and increased artifact size a decent bit without hindering performance if you aren't measuring nanoseconds for a microbenchmark

Reply:
Makes sense. Thanks for the explanation.